Ryujinx.HLE/HOS/Services/Aud/IAudioRendererManager.cs

```diff
@@ -58,8 +58,9 @@ public long GetAudioRendererWorkBufferSize(ServiceCtx context)
 
             if (revision <= Rev)
             {
-                bool isSplitterSupported = revision >= 3;
-
+                bool isSplitterSupported                  = revision >= 3;
+                bool IsVariadicCommandBufferSizeSupported = revision >= 5;
```

Comment:
Lower case `Is` to match the `isSplitterSupported` above it.``
```suggestion
                bool isVariadicCommandBufferSizeSupported = revision >= 5;
```